package.json

```diff
@@ -115,6 +115,7 @@
     "@react-native/assets": "1.0.0",
     "@react-native/normalize-color": "2.1.0",
     "@react-native/polyfills": "2.0.0",
+    "@react-native/virtualized-lists": "1.0.0",
```

Comment:
You should align the version number to React Native's (`0.72.0` in this case).

Reply:
Sure, updated